mod_reforged/hooks/entity/tactical/player.nut

```diff
@@ -37,7 +37,15 @@
 
 	q.getProjectedAttributes <- function()
 	{
-		local baseProperties = this.getBaseProperties();
+		local properties = this.getBaseProperties().getClone();
+
+		local wasUpdating = this.getSkills().m.IsUpdating;
+		this.getSkills().m.IsUpdating = true;
+		foreach (s in this.getSkills().getSkillsByFunction(@( _skill ) _skill.isType(::Const.SkillType.Trait) || _skill.isType(::Const.SkillType.PermanentInjury)))
```

Comment:
There should be a comment somewhere in this area or above the function definition, explaining what we are doing.
Mainly, that the returned attribute array will include changes from permanent injuries and traits